tasks.py.jinja

```diff
@@ -11,21 +11,64 @@ import json
 import os
 import time
 from logging import getLogger
+import tempfile
 from itertools import chain
 from pathlib import Path
 from shutil import which
 
 from invoke import task
+from invoke import exceptions
+from invoke.util import yaml
 
 ODOO_VERSION = {{ odoo_version }}
 PROJECT_ROOT = Path(__file__).parent.absolute()
 SRC_PATH = PROJECT_ROOT / "odoo" / "custom" / "src"
 UID_ENV = {"GID": str(os.getgid()), "UID": str(os.getuid()), "UMASK": "27"}
-SERVICES_WAIT_TIME = int(os.environ.get("SERVICES_WAIT_TIME", 3))
+SERVICES_WAIT_TIME = int(os.environ.get("SERVICES_WAIT_TIME", 4))
 
 _logger = getLogger(__name__)
 
 
+def _override_docker_command(service, command, file, orig_file=None):
+    # Read config from main file
+    if orig_file:
+        with open(orig_file, "r") as fd:
+            orig_docker_config = yaml.load(fd.read())
+            docker_compose_file_version = orig_docker_config.get("version")
+    else:
+        docker_compose_file_version = "2.4"
+    docker_config = {
+        "version": docker_compose_file_version,
+        "services": {service: {"command": command}},
+    }
+    docker_config_yaml = yaml.dump(docker_config)
+    file.write(docker_config_yaml)
+    file.flush()
+
+
+def _remove_auto_reload(file, orig_file):
+    with open(orig_file, "r") as fd:
+        orig_docker_config = yaml.load(fd.read())
+    odoo_command = orig_docker_config["services"]["odoo"]["command"]
+    odoo_command[-1] = odoo_command[-1].replace("reload,", "")
```

Comment:
You can't do this so blindly. Better search the element that starts with `"--dev"` and modify it.